opus/application/static_media/js/browse.js

```diff
@@ -1298,6 +1298,9 @@ var o_browse = {
 
```

Comment:
updateStartobsInUrl is using the current opus.prefs.view. It needs to use the passed-in tab.